system/jhtree/jhtree.cpp

```diff
@@ -660,8 +660,8 @@ class jhtree_decl CKeyLevelManager : implements IKeyManager, public CInterface
             {
                 started = true;
                 numsegs = segs.ordinality();
-                if (numsegs)
-                    segs.checkSize(keyedSize, keyName.get());
+                //if (numsegs)
```

Comment:
Should this be uncommented or removed?

Reply:
It should be put back once I know the keysize and can therefore add the required wildSegMonitors